src/components/editor/EditorArticleController.jsx

```diff
@@ -92,8 +106,55 @@ export default class EditorArticleController extends FalcorController {
     }, 2000);
   }
 
+  // Handle state changes coming from editAuthorsForm
+
+  handleAddAuthor(slug) {
+    // TODO: Autocomplete which also includes validating that the authors indeed exist
+
+    // Is this author already added?
+    if (!slug || this.state.authorsAdded.hasOwnProperty(slug) || this.state.authorsDeleted.hasOwnProperty(slug)) {
```

Comment:
Hmm okay I'll try that, it makes sense for clarity.

It was just because it's two very different functions, but I guess so is my handleDeleteAuthor where I also just pass a parameter to check which function to call inside.